documentation/en/api-v0-methods.md

```diff
@@ -5965,12 +5965,12 @@ Response:
     "InitialPledge": "0",
     "ExpectedDayReward": "0",
     "ExpectedStoragePledge": "0",
-    "ReplacedSectorAge": 10101,
+    "PowerBaseEpoch": 10101,
     "ReplacedDayReward": "0",
     "SectorKeyCID": {
       "/": "bafy2bzacea3wsdh6y3a36tb3skempjoxqpuyompjbmfeyf34fi3uy6uue42v4"
     },
-    "SimpleQAPower": true
+    "Flags": 0
```

Comment:
What change is causing this diff ?

Reply:
This is because we have moved to miner13 types. lgtm.

Reply:
`make gen` generated those after we changed to miner13 in https://github.com/filecoin-project/lotus/pull/11982/commits/941d0befe3b30b910291738e623579ea852c28d9